src/main/java/org/folio/consortia/service/impl/UserServiceImpl.java

```diff
@@ -90,4 +93,13 @@ public User prepareShadowUser(UUID userId, String tenantId) {
     }
   }
 
+  @Override
+  public UserTenant createUserTenant(UserTenant userTenant) {
+    log.info("Creating userTenant with dummy user with id {}.", userTenant.getId());
+    /* Dummy user will be used to support cross-tenant requests checking in mod-authtoken,
+    if user-tenant table contains some record in institutional tenant - it means mod-consortia enabled for
+    this tenant and will allow cross-tenant request. */
```

Comment:
better to add this java doc in place that invokes this method. Because if someone will use this method to create not dummy user - tenant this java doc will be not suitable.
And I think lets not add this method into UserService, I think lets inject userTenantsClient into TenantService

Reply:
fixed